app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GreetingsController < ApplicationController
+  def random
+    greeting = Greeting.order('RANDOM()').first
+    render json: greeting
+  end
```

Comment:
- [ ] Dear @IngDavidTM, I must admit that you have done a great job implementing the functionalities. But as of now, your app has one flaw left to be addressed. which is called `CORS policy`. Because of this, I could not fetch the data from the back end. This happens when the server only replays to clients who have access to use it. [Read more here](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors) or check this block on [how to fix CORS issue on rails](https://www.stackhawk.com/blog/rails-cors-guide/)

https://user-images.githubusercontent.com/69077061/221239343-11c43155-0781-45e5-b329-12fc25651492.mp4

Reply:
Thank you for your comments, the video, and the link. It helped me a lot

Reply:
Nice to hear that @IngDavidTM. Keep up the good work 🙌